packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -46,11 +66,15 @@ type ValidateAttestationDataFn = (attData: phase0.AttestationData) => boolean;
 const MAX_RETAINED_ATTESTATIONS_PER_GROUP = 4;
 
 /**
- * On mainnet, each slot has 64 committees, and each block has 128 (8 in electra) attestations max so in average
+ * Pre-electra, each slot has 64 committees, and each block has 128 attestations max so in average
  * we get 2 attestation per groups.
  * Starting from Jan 2024, we have a performance issue getting attestations for a block. Based on the
- * fact that lot of groups will have only 1 attestation since it's full of participation increase this number
+ * fact that lot of groups will have only 1 full participation attestation, increase this number
  * a bit higher than average. This also help decrease number of slots to search for attestations.
+ *
+ * For electra, each block has up to 8 aggregated attestations, assuming there are 3 for the "best"
+ * attestation data, there are still 5 for other attestation data so this constant is still good.
+ * We should separate to 2 constant based on conditions of different networks
```

Comment:
I agree. Could define a constant named `MAX_ATTESTATIONS_PER_GROUP_ELECTRA`. 

Can add a todo for it

Reply:
added it 👍